src/pages/Dashboard/UsageTiles/CycleUsage-Tile.vue

```diff
@@ -0,0 +1,172 @@
+<script>
+import { formatTime } from '@/mixins/formatTimeMixin'
+import { mapGetters } from 'vuex'
+
+export default {
+  mixins: [formatTime],
+  data() {
+    return {
+      usageLoadingKey: 0,
+      invoiceLoadingKey: 0
+    }
+  },
+  computed: {
+    ...mapGetters('license', ['license']),
+    projectedCost() {
+      if (!this.invoice) return 0
+      return this.invoice.total * 100
+    },
+    nextPaymentDate() {
+      if (!this.invoice) return null
+      const nextPayment = this.invoice.next_payment_attempt * 1000
+      return this.formatLongDate(nextPayment)
+    },
+    periodStart() {
+      if (!this.invoice) return null
+      return new Date(this.invoice.period_start * 1000)
+    },
+    freeUsage() {
+      if (Number.isNaN(this.usage)) return null
```

Comment:
Sometimes when switching between the plans, this.usage would become undefined and I would get "NaN% of free runs used" so I was wondering why we wouldn't use isNaN? 



<img width="465" alt="NaN" src="https://user-images.githubusercontent.com/40726391/113194226-c01c9800-9215-11eb-9ccf-9c715205d9c4.png">

Reply:
Hm good point, I think `isNaN` is more appropriate here.